seed/models/data_quality.py

```diff
@@ -1042,12 +1089,13 @@ def update_status_label(self, label_class, rule, linked_id, row_id):
                     raise IntegrityError(
                         'Label with super_organization_id={} cannot be applied to a record with parent '
                         'organization_id={}.'.format(
+                            label_org_id,
                             label_org_id,
                             taxlot_parent_org_id
                         )
                     )
-
-            self.results[row_id]['data_quality_results'][-1]['label'] = rule.status_label.name
+            if rule.severity != Rule.SEVERITY_VALID:
+                self.results[row_id]['data_quality_results'][-1]['label'] = rule.status_label.name
```

Comment:
I'm thinking we should remove this change. Don't users want to be able to auto apply labels without flagging things as errors?